packages/graphql-language-service-server/src/startServer.js

```diff
@@ -11,117 +11,147 @@
 import net from 'net';
 
 import {
-  processStreamMessage,
-  processIPCRequestMessage,
-  processIPCNotificationMessage,
+  handleDidChangeNotification,
+  handleDidCloseNotification,
+  handleDidOpenOrSaveNotification,
+  handleInitializeRequest,
+  handleCompletionRequest,
+  handleDefinitionRequest,
 } from './MessageProcessor';
 
 import {
   IPCMessageReader,
   IPCMessageWriter,
   SocketMessageReader,
   SocketMessageWriter,
+  StreamMessageReader,
+  StreamMessageWriter,
+  MessageConnection,
+  createMessageConnection,
 } from 'vscode-jsonrpc';
 
+import {
+  DidOpenTextDocumentNotification,
+  DidSaveTextDocumentNotification,
+  DidChangeTextDocumentNotification,
+  DidCloseTextDocumentNotification,
+  ExitNotification,
+  InitializeRequest,
+  CompletionRequest,
+  CompletionResolveRequest,
+  DefinitionRequest,
+  PublishDiagnosticsNotification,
+} from 'vscode-languageserver';
+
 type Options = {
   port?: number,
   method?: string,
   configDir?: string,
 };
 
 export default (async function startServer(options?: Options): Promise<void> {
-  if (!options || !options.configDir) {
+  if (!options) {
+    process.stderr.write('At least --configDir option is required!');
+    process.exit(1);
+    return;
```

Comment:
These `return` statements are for Flow? If so, might be worth asking in the FB Flow group why Flow doesn't know that `process.exit` terminates execution.